openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -381,7 +381,7 @@ public void execute(StfCoreExtension test) throws Exception {
 						.addJarToClasspath(test.env().findPrereqFile(testSuiteFolder + executeJar))
 						.addDirectoryToClasspath(test.env().findPrereqDirectory(testSuiteFolder + "/classes"))
 						.runClass("com.sun.javatest.agent.AgentMain")
-						.addArg(" -passive")
+						//.addArg(" -passive")
```

Comment:
Are the change in this file and the one in makefile intended?

Reply:
No. Updated!